src/sentry/integrations/jira/notify_action.py

```diff
@@ -35,4 +36,23 @@ def render_label(self):
         return self.label.format(name=self.get_integration_name())
 
     def after(self, event, state):
-        pass
+        integration = self.get_integration()
+        installation = integration.get_installation(self.project.organization.id)
+
+        self.data["title"] = event.title
+        self.data["description"] = installation.get_group_description(event.group, event)
+
+        def create_issue(event, futures):
+            """Create the Jira ticket for a given event"""
+
+            # TODO check if a Jira ticket already exists for the given event's issue. if it does, skip creating it
+            resp = installation.create_issue(self.data)
+            ExternalIssue.objects.create(
+                organization_id=self.project.organization.id,
+                integration_id=integration.id,
+                key=resp["key"],
```

Comment:
@ceorourke shouldn't we pass in `defaults` like we do in `src/sentry/api/endpoints/group_integration_details.py`?

Reply:
cool, I didn't see metadata being passed in any other entry so I skipped that now unless we have a need for it